src/nsls2api/models/beamlines.py

```diff
@@ -1,6 +1,6 @@
 import datetime
 from typing import Optional
-
+from beanie import Insert, before_event
```

Comment:
`isort` will probably want to put this after `import pydantic`

Reply:
It did indeed